binaries/cuprated/src/killswitch.rs

```diff
@@ -44,8 +44,8 @@ fn killswitch() {
     /// sanity checking the system's clock to make
     /// sure it is not overly behind.
     ///
-    /// Tue Mar 11 08:33:20 PM UTC 2025
-    const SYSTEM_CLOCK_SANITY_TIMESTAMP: u64 = 1741725200;
+    /// Mon May 12 12:00:00 AM UTC 2025
+    const SYSTEM_CLOCK_SANITY_TIMESTAMP: u64 = 1747008000;
```

Comment:
Am I being silly or should this be April:

 ```suggestion
     /// Tue April 8 12:00:00 AM UTC 2025
     const SYSTEM_CLOCK_SANITY_TIMESTAMP: u64 = 1744070400;
```

Reply:
Oh wow yeah. My bad, not sure how this happened.